program/tests/program_test/cookies.rs

```diff
@@ -443,31 +390,29 @@ impl SpotMarketCookie {
 
         let admin_pk = test.get_payer_pk();
 
-        let instructions = [
-            mango::instruction::add_spot_market(
-                &mango_program_id,
-                &mango_group_pk,
-                &spot_market_cookie.market,
-                &serum_program_id,
-                &test.mints[mint_index].pubkey.unwrap(),
-                &node_bank_pk,
-                &vault_pk,
-                &root_bank_pk,
-                &admin_pk,
-                mint_index,
-                maint_leverage,
-                init_leverage,
-                liquidation_fee,
-                optimal_util,
-                optimal_rate,
-                max_rate,
-            ).unwrap(),
-        ];
+        let instructions = [mango::instruction::add_spot_market(
+            &mango_program_id,
+            &mango_group_pk,
+            &oracle_pks[mint_index],
```

Comment:
here's the actual diff

i tried first to get `mango_group_cookie.mango_group.oracles[mint_index]` but the value was stale. if there was a way to refresh the group this would be basically a two-line change (also need to remove mint_index) rather than a whole puncher through all layers.